src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+
+const promise1 = new Promise(firstCallback);
+
+function firstCallback(complete, eror) {
```

Comment:
bettre use resolve, reject for this function as in previous comment. This is more common